gitrevise/odb.py

```diff
@@ -308,7 +308,7 @@ def get_obj(self, ref: Union[Oid, str]) -> "GitObj":
         self._catfile.stdin.flush()
 
         # Read in the response.
-        resp = self._catfile.stdout.readline().decode("ascii")
+        resp = self._catfile.stdout.readline().decode("utf-8")
```

Comment:
I think this change is also good, as if a non-ascii ref is passed in, the `missing` line could contain non-ascii characters.

Reply:
Good point. Resliced to put that in the first commit.